build/meson/meson.build

```diff
@@ -16,7 +16,7 @@ project('zstd',
     'cpp_std=c++11',
     'buildtype=release'
   ],
-  version: '1.3.8',
+  version: '1.4.0',
```

Comment:
Same comment (as the `lz4` PR) regarding the version number

Reply:
This is fallback version number in case of extracting version number from
`zstd.h` failed.

Reply:
Is it possible to always use the version in `zstd.h` and fail the build if the extraction fails? That would be preferable, because then we don't have to remember to update the meson build file.

Reply:
Done and done.